File: Problems/2482. Difference Between Ones and Zeros in Row and Column/2482. Difference Between Ones and Zeros in Row and Column.java
class Solution {
    public int[][] onesMinusZeros(int[][] grid) {
        int m = grid.length;
        int n = grid[0].length;
        int[] onesRow = new int[m];
        int[] onesCol = new int[n];

        for (int i = 0; i < m; i++) {
            for (int j = 0; j < n; j++) {
                onesRow[i] += grid[i][j];
                onesCol[j] += grid[i][j];
            }
        }

        for (int i = 0; i < m; i++)
            for (int j = 0; j < n; j++)
                grid[i][j] = (2*onesRow[i]-n) + (2*onesCol[j]-m);

        return grid;
    }
}

File: Problems/2482. Difference Between Ones and Zeros in Row and Column/2482. Difference Between Ones and Zeros in Row and Column.py
from itertools import product;
class Solution:
    def onesMinusZeros(self, grid: List[List[int]]) -> List[List[int]]:
        m, n = len(grid), len(grid[0]);
        onesRow = [row.count(1) for row in grid];
        onesCol = [col.count(1) for col in zip(*grid)];

        for i, j in product(range(m), range(n)):
            grid[i][j] = (2*onesRow[i]-n) + (2*onesCol[j]-m);

        return grid;